contents/blog/startup-data-stack.md

```diff
@@ -0,0 +1,108 @@
+
+Moving from individual sources to a CDP makes gathering data faster, but starts to create a web of integrations and mainteinance. It is a challenge to know if data is accurate, luckily at this stage, it doesn't need to be. The Seed stage strikes the balance of giving engineers enough data to know what to prioritize with having plenty of time to build it.
+
+## The Series A data stack
+
+Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the "modern data stack" which contains four tools:
+
+1. A data warehouse to store all the data and act as a single source of truth. Options include Snowflake, BigQuery, and RedShift.
+2. An ETL pipeline to extract data from different sources, transform it, and load it into the warehouse. Options include Fivetran, Integrate.io, and Airbyte.
+3. A data transformation tool to model data, clean it up, and make it usable. dbt is basically the only option.
+4. A business intelligence or visualization tool to get insights from the data you’ve collected. Options include Metabase, Looker, and Hex.
+![Series A](../images/blog/startup-data-stack/series-a.png)
+
+Startups need this stack because managing and accessing individual sources of data becomes unsustainable. Tools like analytics and the CDP don't give complete insight into performance. Teams need a single source of truth, and the modern data stack gives them that. This creates trust in the data and greater accuracy.
+
+The downside of this stage is that a gap opens between engineers and the data that is valuable to them. They are unable to "self-serve" and must learn the modern data stack tools or rely on the data team for insights. There are many reasons for this:
+
+- The complexity of data and tools requires specialization.
+- Data security, safety, and privacy requirements.
+- Lack of knowledge of data available or how to use it.
+
+For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. There all create friction on what engineers want to do: build better products. 
+
+## PostHog’s vision for the future of the startup data stack
+
+Being a part of this evolution, we have thoughts on where PostHog fits into this. [Our mission](/handbook/strategy/overview) is to increase the number of successful products in the world, and we do this by helping engineers be better at product. This means helping them gather and analyze data at the earliest stage of their product, and integrating more of the parts of the startup data stack as they scale up.
```

Comment:
I think we should frame this opening a bit more aggressively – i.e. make it "The problem with the 'modern data stack' and how we want to fix it" and open with a more opinionated take why the current norms are bad. All the right bits of information is here, it just feels a bit polite. I think we should be confident enough to say "this sucks" and here's a better way.

Reply:
got it I think